README.md

```diff
@@ -9,31 +9,155 @@ is [Disnake](https://github.com/DisnakeDev/disnake/).
    cd SincereSingularities
    pip install -e .
    ```
-4. Set the `BOT_TOKEN` environment variable to your Token using the `.env` file.
-5. Run The Game:
+4. Setup and run [MongoDB Community Edition](https://www.mongodb.com/docs/manual/administration/install-community/)
+5. Set the `BOT_TOKEN`, `DB_HOST`, `DB_PORT` and `DB_NAME` environment variables using the `.env` file.
+6. Run The Game:
    ```shell
    python -m sincere_singularities
    ```
-6. Play!
+7. Play!
+
+## Video Presentation
+
+Disclaimer: `The video contains Spoilers`
+
+<details>
+    <summary>Video Presentation / Description</summary>
+    https://www.youtube.com/
+</details>
+
+
+## Installation
+<details>
+    <summary>Detailed Installation Guide ⚙️</summary>
+
+### 1. Requirements:
+   1. [Python 3.11](https://www.python.org/downloads/release/python-3110/)
+   2. [MongoDB Community Edition](https://www.mongodb.com/docs/manual/administration/install-community/). See [MongoDB Installation Guide]
+   3. [Discord Bot](https://docs.disnake.dev/en/stable/discord.html). See [Discord Bot SetUp Guide]
+### 2. Download:
+Run this command in the directory you want to download it to.
+   ```shell
+   git clone https://github.com/SincereSingularities/SincereSingularities/
+   cd SincereSingularities
+   ```
+### 3. Install the Game as a PIP package & install requirements:
+   ```shell
+   pip install -e .
+   ```
+### 4. Setup local environment values:
+Create & Edit an .env file (see .env.example)
+```
+BOT_TOKEN (Your Discord Bot Token)
+DB_HOST (The IP address of your MongoDB Server)
+DB_PORT (The port of your MongoDB Server)
+DB_NAME (Your preffered name for the MongoDB Database)
+
+```
+   
+</details>
+
+## Running
+<details>
+    <summary>Detailed Running Guide ⚙️</summary>
+
+### 1. Start your [MongoDB Server](https://www.mongodb.com/docs/manual/administration/install-community/). See [MongoDB Installation Guide]
+### 2. Run the Game:
+   ```shell
+   python -m sincere_singularities
+   ```
+### 3. Start a Game Session in a Text Channel:
+   ```
+   /start_game
+   ```
+</details>
 
 ## Gameplay
+<details>
+    <summary>Gameplay Explanation 💡</summary>
 
 1. Choose a text channel.
 2. Run the `/start_game` command. That will create a thread.
 3. The bot will send the menu. This contains multiple restaurants that you can buy. You already own the first one.
 4. You will get orders in the thread as messages. Choose and enter the appropriate restaurant, and select the menu items that the customer requested.
 5. Then enter the customer's information which consists of:
+- Order ID
+- Customer Name
+- Customer Address
+- Delivery Time
+- Extra Wishes
+6. Enter the ordered items.
+7. Submit the order. You will get points based on the accuracy of the order. With your earned coins you can buy new restaurants.
+8. From time to time, you will get Order Conditions. Pay attention to these Conditions when fulfilling a order. They will also disappear after some time.
```

Comment:
Better explanation maybe? What exactly are they? What should you do? They only affect certain restaurants. When the message disappears, it's no longer active. ...

Reply:
IMO, the Gameplay Explanation should only explain the game in its basis. If you want to learn how the Order Conditions actually work, either play the game yourself (learning-by-doing) or whatch the video presentation.

Reply:
Ok. :+1: